.env.sample

```diff
@@ -50,3 +50,21 @@ ATLAS_DOMAIN_NAME=https://atlas.cosmos.network
 
 # The syslog destination address for log streaming.
 ATLAS_SYSLOG_ADDR=logsN.logs.com:XXXX
+
+# The ipstack API key for geolocation functionality.
+ATLAS_IPSTACK_KEY = "013d2f2737f1353bc000f1ed4d61ed44"
+
+# The interval in which to retrigger a node crawl after the node pool is
+# exhausted.
+ATLAS_NODE_CRAWL_INTERVAL = "5m"
+
+# The interval in which to recheck nodes for availability.
+ATLAS_NODE_RECHECK_INTERVAL = "1h"
+
+# The size of the reseed list for which to reseed the node pool after it has
+# been exhausted. It is populated by the crawling process.
+ATLAS_NODE_RESEED_SIZE = 100
+
+# The initial list of comma-delimited seed nodes to crawl, where each seed takes
+# the form of [host]:[port];[chain-id], where ;[chain-id] is optional.
+ATLAS_NODE_SEEDS = "http://1.255.51.125:26657;cosmsohub-3,..."
```

Comment:
would be great to target https://github.com/cosmos/registry eventually and in the future when these get moved to the [cosmos name service](https://github.com/tendermint/spn/pull/100) to query them there (oh shit what a chicken or egg problem tho, need a node to query the node for the list of nodes 🤪)